src/components/feed-display/FeedItemRow.vue

```diff
@@ -8,22 +8,31 @@
 				:href="item.url"
 				:title="t('news', 'Open website')"
 				@click="markRead(item); $event.stopPropagation();">
-				<EarthIcon />
+				<span v-if="getFeed(item.feedId).faviconLink"
+					class="favicon"
+					:style="{ 'backgroundImage': 'url(' + getFeed(item.feedId).faviconLink + ')' }" />
+				<RssIcon v-else />
 			</a>
-			<RssIcon v-if="!getFeed(item.feedId).faviconLink" />
-			<span v-if="getFeed(item.feedId).faviconLink" style="width: 24px; background-size: contain;" :style="{ 'backgroundImage': 'url(' + getFeed(item.feedId).faviconLink + ')' }" />
 		</div>
-		<div class="title-container" :class="{ 'unread': item.unread }">
-			<span style="white-space: nowrap" :dir="item.rtl && 'rtl'">
-				{{ item.title }}
+
+		<div class="main-container">
+			<div class="title-container" :class="{ 'unread': item.unread }">
+				<span style="white-space: nowrap" :dir="item.rtl && 'rtl'">
```

Comment:
should this style be moved to a css rule?

Reply:
This was just moved, I did not change/add it.. but I can move it into a class if you like.. 😉